DataProcessor/DataUnpacker.cpp

```diff
@@ -54,12 +54,16 @@ DataUnpacker::DataUnpacker(QObject *parent) : QObject(parent)
     int dataCount = 0;
     cell_group_voltages_begin = -1;
     cell_group_voltages_end = -1;
+#ifdef FIRMWARE
+    controlsOffsets offsets;
+#endif
 
     for(Value::ConstMemberIterator itr = d.MemberBegin(); itr != d.MemberEnd(); ++itr) {
         std::string name = itr->name.GetString();
         const Value& arr = itr->value.GetArray();
 
         names.push_back(name);
+        qDebug() << QString::fromStdString(name);
```

Comment:
Might be for another PR, but since we have various firmware and software printouts that get in the way of each other, it might be nice to have a generic utils or debug file that defines a custom debug function, or multiple debug functions if we wanted to use `printf()` vs `qDebug()` for driver vs software development, to allow us to conditionally enable debugging for certain features/pieces of code. I was thinking we could have flags/definitions like `TCA_DEBUG`, `NETWORK_DEBUG`, etc. and then have (a) function(s) along the lines of
```
void firmwareDebug(const char * message, int print) {
  if (print)
    printf(message);
}

/* Call example */
firmwareDebug("Print this message conditionally", TCA_DEBUG);
```